openid-connect-server-spring-boot-test/src/test/java/org/mitre/boot/security/oidc/JwtAuthorizeEndpointFlowTests.java

```diff
@@ -224,7 +224,7 @@ public void codeResponseTypeLoginFlowWhitelistTest() throws Exception{
 
     protected void validateAccessToken(final SignedJWT accessToken, final String clientId) throws ParseException{
         Assert.assertEquals(clientId, accessToken.getJWTClaimsSet().getAudience().get(0));
-        Assert.assertEquals("http://localhost:-1/", accessToken.getJWTClaimsSet().getIssuer());
+        Assert.assertEquals("http://localhost:8080/", accessToken.getJWTClaimsSet().getIssuer());
```

Comment:
testin application.ymlissä on serverin issuerissa server.port portissa ja se on 8080, tuntus että tämä on näin oikeampi kuin aiemmin oli?

Reply:
joo. tuo -1 ei ainakaan kuulosta järkevältä